manager/integration/deploy/test.yaml

```diff
@@ -94,6 +94,14 @@ spec:
     - name: longhorn
       mountPath: /var/lib/longhorn/
       mountPropagation: Bidirectional
+    - name: test-report
```

Comment:
```suggestion
    - name: longhorn-test-report
```

Reply:
Remember to update other places accordingly as well.

Reply:
sorry but just want to confirm something:
rename container name "keepalive" to "longhorn-test-report"
or
rename volume name "test-report" to "longhorn-test-report" ?

Reply:
yeah, container name, I think I marked the wrong place :)